### PR TITLE
Query NOC Intermediate by VID+SKID

### PR DESCRIPTION
**Pull request by DenisRybas**

### CODE REVIEW

File: integration_tests/grpc_rest/pki/noc_cert_helpers.go [hunk 1]
```diff
@@ -106,29 +106,29 @@ func GetNocX509RootCerts(suite *utils.TestSuite, vendorID int32) (*pkitypes.NocR
 	return &res, nil
 }
 
-func GetNocX509RootCertsByVidAndSkid(suite *utils.TestSuite, vendorID int32, subjectKeyID string) (*pkitypes.NocRootCertificatesByVidAndSkid, error) {
-	var res pkitypes.NocRootCertificatesByVidAndSkid
+func GetNocX509RootCertsByVidAndSkid(suite *utils.TestSuite, vendorID int32, subjectKeyID string) (*pkitypes.NocCertificatesByVidAndSkid, error) {
```
Comment (ashcherbakov) [inline integration_tests/grpc_rest/pki/noc_cert_helpers.go:109]: Probably need to rename the method as well (get rid of `root` in the name)

File: x/pki/types/key_noc_root_certificates_by_vid_and_skid.go [hunk 2]
```diff
@@ -5,12 +5,12 @@ import "encoding/binary"
 var _ binary.ByteOrder
```
Comment (ashcherbakov) [inline x/pki/types/key_noc_root_certificates_by_vid_and_skid.go:6]: Should we rename the file (get rid of `root`)?

**Review by ashcherbakov — COMMENTED** [review]
Should we also update https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/scripts/starport/upgrade-0.44/07.pki_types.sh and/or https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/scripts/ignite/07.pki_types.sh?

File: x/pki/types/key_noc_root_certificates_by_vid_and_skid.go (hunk 2 above)
Comment (DenisRybas) [reply x/pki/types/key_noc_root_certificates_by_vid_and_skid.go:6]: Fixed

File: docs/transactions.md [hunk 3]
```diff
@@ -1525,24 +1525,24 @@ Use [GET_ALL_REVOKED_NOC_ROOT](#get_revoked_noc_root) to get a list of all revok
 - REST API:
   - GET `/dcl/pki/noc-root-certificates/{vid}`
 
-#### GET_NOC_ROOT_BY_VID_AND_SKID
+#### GET_NOC_BY_VID_AND_SKID
 
 **Status: Implemented**
 
-Retrieve NOC root certificates associated with a specific VID and subject key ID.
+Retrieve NOC certificates associated with a specific VID and subject key ID.
```
[the remark this answers is not in the document]
Comment (DenisRybas) [reply docs/transactions.md]: Clarified

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**